app/components/SCForm.js

```diff
@@ -18,6 +19,8 @@ import { buttonStyles } from '../style/style';
 transform.registerType('date', tcomb.Date);
 transform.registerType('time', tcomb.Date);
 
+const t = require('tcomb-validation');
+const validate = t.validate;
```

Comment:
use the es6 import here http://exploringjs.com/es6/ch_modules.html#sec_importing-exporting-details